tasks/st0317a_clean_cert.rb

```diff
@@ -25,7 +29,7 @@ def pe_primary?
 ca_cfg = pe_primary? ? '/etc/puppetlabs/puppetserver/bootstrap.cfg' : '/etc/puppetlabs/puppetserver/services.d/ca.cfg'
 
 if !File.exist?(ca_cfg) || File.readlines(ca_cfg).grep(%r{^[^#].+certificate-authority-service$}).empty?
-  puts 'This task can only be run on your certificate authority Puppetserver'
+  puts '***THIS TASK IS NOW DEPRECATED - Please see README*** - This task can only be run on your certificate authority Puppetserver'
```

Comment:
I don't really like adding this message to everything the script may output.  We should add it to the README and the main results, but I think we should remove this one.